.github/workflows/continuous-integration.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           - os: ubuntu-latest
             java: 21
           - os: ubuntu-latest
-            java: 22
```

Comment:
We don't need to do long-term testing on non-LTS releases so just switch 22 to 23